node/pegnet/addresses.go

```diff
@@ -71,11 +71,105 @@ const createTableAddresses = `CREATE TABLE IF NOT EXISTS "pn_addresses" (
         "pzec_balance"  INTEGER NOT NULL DEFAULT 0
                         CONSTRAINT "insufficient balance" CHECK ("pzec_balance" >= 0),
         "pdcr_balance"  INTEGER NOT NULL DEFAULT 0
-                        CONSTRAINT "insufficient balance" CHECK ("pdcr_balance" >= 0)
+                        CONSTRAINT "insufficient balance" CHECK ("pdcr_balance" >= 0),
+        -- v4 additions
+        "paud_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("paud_balance" >= 0),
+        "pnzd_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("pnzd_balance" >= 0),
+        "psek_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("psek_balance" >= 0),
+        "pnok_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("pnok_balance" >= 0),
+        "prub_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("prub_balance" >= 0),
+        "pzar_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("pzar_balance" >= 0),
+        "ptry_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("ptry_balance" >= 0),
+        "peos_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("peos_balance" >= 0),
+        "plink_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("plink_balance" >= 0),
+        "patom_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("patom_balance" >= 0),
+        "pbat_balance"  INTEGER NOT NULL DEFAULT 0
+                        CONSTRAINT "insufficient balance" CHECK ("pbat_balance" >= 0),
+        "pxtz_balance"  INTEGER NOT NULL DEFAULT 0 CHECK ("pxtz_balance" >= 0)
```

Comment:
missing constraint for pxtz_balance

Reply:
I had the check, but not the named constraint. Whoops.